spec/lib/decision_review/service_spec.rb

```diff
@@ -195,4 +197,28 @@
       end
     end
   end
+
+  describe '#get_contestable_issues' do
+    context 'with a valid constestable issues request' do
+      it 'returns a contestable issues response object' do
+        VCR.use_cassette('decision_review/200_contestable_issues') do
+          response = subject.get_contestable_issues(user)
+          expect(response).to be_ok
+          expect(response).to be_an DecisionReview::Responses::Response
```

Comment:
Maybe add verification of the response body here?

Reply:
Thanks for the review Lindsey! The reason I dont validate the response payload is because theres a json schema validation built into the response object. So to verify it again seemed redundant to me.